### PR TITLE
Updating to readthedocs format, adding .vscode to gitignore.

### PR DESCRIPTION
**Pull request by giovannicuriel**

### CODE REVIEW

File: docs/profile-configuration.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
+ejbca-rest
+==========
+
+Configuration
+-------------
+
+If you are using the provided docker container, all configuration
+explained on this document was automatic done on the container.
```
Comment (mmagr) [inline docs/profile-configuration.rst]: > explained on this document was **automatically** done on the container.

or perhaps?

> explained on this document has already been done on the container.

File: docs/profile-configuration.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+This document list the steps taken to manually configure EJBCA for
```
Comment (mmagr) [inline docs/profile-configuration.rst]: > This document **lists** the steps taken to ~manually~ configure EJBCA. ~for~
> ~historic and the sake of transparency on the configuration done.~

File: docs/profile-configuration.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+historic and the sake of transparency on the configuration done.
+
+Misconfigure a Certificate Authority can be dangerous. Please, take
```
Comment (mmagr) [inline docs/profile-configuration.rst]: > **Misconfiguring** a Certificate Authority can be dangerous. Please, take

Or perhaps?

> Misconfiguring a Certificate authority can severely affect all systems that rely on it for authentication purposes, leading to a swath of security vulnerabilities being implicitly added to them. Please be extra careful when dealing with this.

File: docs/profile-configuration.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+care.
+
+Pre Configuration
+~~~~~~~~~~~~~~~~~
+
+To configure EJBCA via the web interface, first you need to configure
+your browser certificates.
+
+Copy the client side certificate from you EJBCA instance, located on
```
Comment (mmagr) [inline docs/profile-configuration.rst]: > Copy the client side certificate from **your** EJBCA instance, located on

File: docs/profile-configuration.rst [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+'/p12/superadmin.p12' and '/p12/ca.crt' to your local machine.
+
+How to configure certificates vary from browser to browser. Please,
```
Comment (mmagr) [inline docs/profile-configuration.rst]: perhaps?
> The way trusted and client certificates are configured vary from browser to browser. Please,

File: docs/profile-configuration.rst (hunk 1 above)
Comment (giovannicuriel) [reply docs/profile-configuration.rst]: Ok

File: docs/profile-configuration.rst (hunk 2 above)
Comment (giovannicuriel) [reply docs/profile-configuration.rst]: I've already fixed this somewhere else.... but ok.

File: docs/profile-configuration.rst (hunk 3 above)
Comment (giovannicuriel) [reply docs/profile-configuration.rst]: Ok

File: docs/profile-configuration.rst (hunk 5 above)
Comment (giovannicuriel) [reply docs/profile-configuration.rst]: Ok.

File: docs/profile-configuration.rst (hunk 4 above)
Comment (giovannicuriel) [reply docs/profile-configuration.rst]: Ok.

**Pull request merged**